_schemaV2.graphql

```diff
@@ -701,7 +701,6 @@ type Artist implements Node & Searchable & EntityWithFilterArtworksConnectionInt
   isFollowed: Boolean
   isPublic: Boolean
   isShareable: Boolean
-  isTargetSupply: Boolean
```

Comment:
not sure why this keeps getting removed

Reply:
It's not actually in the .ts schema definitions.